sdk/trace/provider.go

```diff
@@ -182,6 +182,19 @@ func (p *TracerProvider) ApplyConfig(cfg Config) {
 	p.config.Store(&c)
 }
 
+func (p *TracerProvider) Shutdown() {
```

Comment:
Please add a docblock and tests for this method.

Reply:
Added

Reply:
It looks like most of the `TracerProvider` methods are covered incidentally in other tests.  This method would require a test of its own, so creating a new `provider_test.go` file and adding the test there would probably be appropriate.

Reply:
@Aneurysm9 I added `provider_test.go`. Please help to take a look. Thanks.